draft-normington-jsonpath-latest.xml

```diff
@@ -384,50 +384,115 @@ array-slice = [ integer ] ws %x3A ws [ integer ]
                   </section>
                   <section numbered="false" toc="exclude">
                      <name>Semantics</name>
-                     <t>An array index selects any element of an array which is indexed by the integer index.
-                        It selects no nodes from a node which is not an array.
```

Comment:
The behaviour of array access for non-array nodes is now undefined.

Reply:
good catch; I put it back where `a(i)` is defined

Reply:
I can see it in the syntax section, but it's probably better to move that to somewhere near the top of the semantics section.

Reply:
moved on top